### ISSUE TITLE
Native Notebooks: Can't figure out how to start kernel

### ISSUE THREAD

**Issue by nickeubank** [microsoft/vscode-jupyter#6568]
## Environment data

-   VS Code version:  1.57.1
-   Jupyter Extension version (available under the Extensions sidebar): v2021.6.999662501
-   Python Extension version (available under the Extensions sidebar): v2021.6.944021595
-   OS (Windows | Mac | Linux distro) and version: 11.4
-   Python and/or Anaconda version: n/a
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): n/a
-   Jupyter server running: Local

Turned on experiment with:

```jsonf
    "workbench.editorAssociations": {
        "*.ipynb": "jupyter-notebook"
    },
    "jupyter.experiments.optInto": [
        "__NativeNotebookEditor__"
    ],
```

## Expected behaviour

I opened up a new blank native notebook, but can't seem to get a kernel running.

## Actual behaviour

![no_kernel](https://user-images.githubusercontent.com/9683693/124816268-4db70480-df36-11eb-81a6-50656b28ad3a.gif)

## Logs

Log doesn't change through this process.

**Comment by joyceerhl** [comment]
The kernel picker is empty, which suggests that we failed to register any Jupyter kernels with VS Code. Do you happen to have Python interpreters and/or kernels installed locally?

**Comment by nickeubank** [comment]
Many. The other oddity is that the kernel selection button wasn't appearing in the top right.

Unfortunately, about an hour later I realized everything was working again -- the kernel selection button had returned, and that list is populated. The only thing that I think I did in the interim was play around with the Cell Toolbar setting, but... I've tried to put stuff back to where it was before I did anything, and I can't get the bug to return. So... this may be one of those intermittent problems I don't know how to recreate. Sorry!
